routes/product.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const router = express.Router();
+
+const ProductService = require('../services/ProductService');
+const ProductServiceInstance = new ProductService();
+
+module.exports = (app) => {
+    app.use('/product', router);
+
+    router.get('/', async(req, res, next) => {
+        try {
+            const queryParams = req.query;
+
+            const response = await ProductServiceInstance.list(queryParams);
+            res.status(200).send(response)
+        } catch(err) {
+            next(err);
+        }
+
+    })
+
+    router.get('/:productId', async(req, res, next) => {
```

Comment:
kada dodas i ostale rute, trebalo bi dodati i komentare za swagger kao u onom testnom primjeru (u routes/order.js datoteci).. 

napravi ove rute prvo, pa to mozes nakon... kad napravis rute cu ja povuc si branch i pobrinut se da swagger radi..